test/Java11andUp/src/org/openj9/test/java_lang/Test_String.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package org.openj9.test.java_lang;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.log4testng.Logger;
+
+/**
+ * This test Java.lang.String API added in Java 11 and later version.
+ *
+ */
+public class Test_String {
+	public static Logger logger = Logger.getLogger(Test_String.class);
+
+	/*
+	 * Test Java 11 API String.valueOfCodePoint(codePoint)
+	 */
+	@Test(groups = { "level.sanity" })
+	public void testValueOfCodePoint() {
+		try {
+			Method method = String.class.getDeclaredMethod("valueOfCodePoint", int.class);
+			method.setAccessible(true);
+			// Verify that for each valid Unicode code point i,
+			// i equals to String.valueOfCodePoint(i).codePointAt(0).
+			for (int i = Character.MIN_CODE_POINT; i <= Character.MAX_CODE_POINT; i++) {
+				try {
+					String str = (String) method.invoke(null, i);
+					Assert.assertEquals(i, str.codePointAt(0), "Unicode code point mismatch at " + i);
+				} catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
```

Comment:
Test cases shouldn't catch unexpected exceptions.  Let the test code throw them and the test harness will (usually) print the relevant stacktrace.

Reply:
Fixed, removed catching of unexpected exceptions.